libraries/helpers.rb

```diff
@@ -255,6 +255,14 @@ def get_config(product)
       node.run_state[:ingredient_config_data] ||= {}
       node.run_state[:ingredient_config_data][product] ||= ''
     end
+
+    def fqdn_resolves?(fqdn)
+      require 'resolv'
+      Resolv.getaddress(fqdn)
+      return true
+    rescue
+      false
+    end
```

Comment:
You might consider using `Module#module_function` to make these sort of functions callable with the Module as the receiver:

```ruby
module_function :fqdn_resolves?
```

That would allow the following:

```ruby
ChefIngredientCookbook::Helpers.fqdn_resolves?(fqdn)
```